slack.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Slack
+permalink: /slack
+tags: about
+---
+
+> Want an invite? Anyone who donates to the [Soft Skills Engineering Patreon](https://www.patreon.com/SoftSkillsEng) gets an invite to the Slack.
+
+Welcome to the Soft Skills Engineering Slack! We're excited to have you join this community interested in the non-technical parts of the technical field of software. These guidelines will help keep the environment safe, welcoming, and friendly.
+
+## Guidelines
+1. Hop in to #intros and introduce yourself. Help us get to know you!
+2. Fill out your profile. Make yourself a human.
+3. Read the [Code of Conduct](/code-of-conduct).
+4. Remmeber that everyone is welcome to contribute regardless of experience level or background. If you just want to hang back and lurk, that’s cool too.
+5. This Slack is not a place for commercial activity (e.g. recruiting, or marketing your product/conference) except in channels dedicated to that purpose (e.g. #jobs, #consulting-gigs, #conferences, or #services) and never in an unsolicited direct message. If you see commercial behavior going down, direct message @jamison and I’ll fix it.
```

Comment:
Same comment as above. We don't have these channels (yet?), and I'm inclined to keep the channel count down for simplicity, at least at first. What do you think?